Controller/Component/WizardComponent.php

```diff
@@ -382,7 +382,7 @@ public function process($step) {
 					} else {
 						throw new NotImplementedException(sprintf(__('Process Callback not found. Please create Controller::%s', $processCallback)));
 					}
-					if ($proceed) {
+					if ($proceed === true) {
```

Comment:
That specifically talks about falsey values if they represent different things than null.

Again, I think something is wrong with your unit test. I need more context on how this code is called.